roadmap.md

```diff
@@ -1,192 +1,70 @@
 ---
-title: TiDB v4.0 Roadmap
-summary: Learn about the v4.0 roadmap of TiDB.
-aliases: ['/docs/dev/roadmap/','/docs/ROADMAP/','/docs/roadmap/']
+title: TiDB Roadmap
+aliases: ['/docs/ROADMAP/','/docs/roadmap/']
 ---
 
 <!-- markdownlint-disable MD001 -->
 
-# TiDB v4.0 Roadmap
+# TiDB Roadmap
 
-This document describes the roadmap for TiDB development.
+## Improve system stability
 
-## TiDB
+- [ ] Create bindings for `UPDATE`/`DELETE`/`INSERT` queries.
+- [ ] Optimize the pessimistic transaction model when DDL and DML statements are executed at the same time to improve system stability.
+- [ ] Reduce latency jitter.
 
-### TiDB Server
+## Improve system performance and reduce latency
 
-#### Features
+- [ ] Optimize the performance and efficiency of bulk deletion.
+- [ ] Improve memory management.
+- [ ] Improve the accuracy and robustness of index selection.
+- [ ] Improve the performance of partition pruning and data access on the partitioned table.
+- [ ] Asynchronous commit.
+- [ ] Clustered Index.
```

Comment:
```suggestion
- [ ] Clustered Primary Index.
```

Reply:
Clustered Index is a proper noun, which needs to be reinterpreted after revision.